### PR TITLE
nodejs-es: Sync site.json with english version

### PR DESCRIPTION
**Pull request by Tsuna-mi**
This PR update locale/es/site.json file as its english version.
Thanks for reviewing it! :)

Related with #1778 
cc @nodejs/nodejs-es

### CODE REVIEW

File: locale/es/site.json [hunk 1]
```diff
@@ -88,22 +99,30 @@
...
       "text": "Guías"
+    },
+    "dependencies": {
+      "link": "docs/meta/topics/dependencies",
+      "text": "Dependencias"
     }
   },
   "getinvolved": {
     "link": "get-involved",
-    "text": "Participa",
```
Comment (academo) [inline locale/es/site.json]: I think "Participa" makes more sense in this context rather than "Implícate"

Comment (Tsuna-mi) [reply locale/es/site.json]: I tried it with 'implícate' cause is the word used at catalan translation. I can change in both files by 'Participa' to be consistent. 
Thanks @academo

**Review by franher — APPROVED** [review]
LGTM :+1:

**Comment by fhemberger** [comment]
@Tsuna-mi Could you please check for merge conflicts? Would like to get this merged.

**Comment by Tsuna-mi** [comment]
> @Tsuna-mi Could you please check for merge conflicts? Would like to get this merged.

@fhemberger  Sorry I don't see conflicts.. I've fixed last week
captura de pantalla 2018-10-22 a las 18 14 08

**Comment by fhemberger** [comment]
@Tsuna-mi Ok, may be related to GitHub's hiccup today. Please go ahead and merge this PR, I can't do it myself:

bildschirmfoto 2018-10-22 um 18 37 09

**Pull request merged**